pkg/api/server/register_containers.go

```diff
@@ -1463,5 +1490,34 @@ func (s *APIServer) registerContainersHandlers(r *mux.Router) error {
 	//   500:
 	//     $ref: "#/responses/InternalError"
 	r.HandleFunc(VersionedPath("/libpod/containers/{name}/init"), s.APIHandler(libpod.InitContainer)).Methods(http.MethodPost)
+	// swagger:operation POST /libpod/containers/{name}/rename renameContainer
```

Comment:
```suggestion
	// swagger:operation POST /libpod/containers/{name}/rename libpod libpodRenameContainer
```
Try this. I have not tested it but I am confident that this fixes the swagger test.